Emby.Dlna/PlayTo/Device.cs

```diff
@@ -762,6 +758,43 @@ private async Task<(bool, uBaseObject)> GetPositionInfo(TransportCommands avComm
             return (true, uTrack);
         }
 
+        private XElement ParseResponse(string xml)
+        {
+            // Handle different variations sent back by devices
+            try
+            {
+                return XElement.Parse(xml);
+            }
+            catch (XmlException)
+            {
+
```

Comment:
shouldn't we log at least a debug message here and in other places where hacks over xml were tried?